src/main/java/com/stuypulse/robot/commands/DrivetrainGoalCommand.java

```diff
@@ -10,14 +10,18 @@
 /** Command that aligns with the target */
 public class DrivetrainGoalCommand extends DrivetrainAlignmentCommand {
 
+    public DrivetrainGoalCommand(Drivetrain drivetrain, DrivetrainGoalAligner goalAligner) {
+        super(drivetrain, goalAligner);
+        setMaxSpeed(Constants.Alignment.Speed.LIMELIGHT_MAX_SPEED);
+        useMinTime();
+    }
+
     /**
      * @param drivetrain drivetrain to do movements on
      * @param distance distance to move from the target
      * @param innerGoal if we should aim for the inner goal
      */
     public DrivetrainGoalCommand(Drivetrain drivetrain, Number distance) {
-        super(drivetrain, new DrivetrainGoalAligner(distance));
-        setMaxSpeed(Constants.Alignment.Speed.LIMELIGHT_MAX_SPEED);
-        useMinTime();
+        this(drivetrain, new DrivetrainGoalAligner(distance));
```

Comment:
`setMaxSpeed(Constants.Alignment.Speed.LIMELIGHT_MAX_SPEED);`
and
`useMinTime();`

are very important to this class aligning correctly.

Reply:
Those functions are called within the other constructor